docs/howto/upgrade-cluster/aws.md

```diff
@@ -119,10 +128,32 @@ cluster is unused or that the maintenance is communicated ahead of time.
    export AWS_SECRET_ACCESS_KEY="..."
    ```
 
-3. *Upgrade the k8s control plane's one minor version*
+### 4. Upgrade the k8s control plane's one minor version for three times in a row
```

Comment:
```suggestion
### 4. Upgrade the k8s control plane's minor version incrementally through consecutive versions
```

I'm trying to generalise this statement a bit, because it will depend on what the starting version is and what the desired version is as to whether it's two, three, or four times!

Reply:
Reading further on, I see we cannot have node groups more than 3 minor versions behind, so maybe this should be:

```
Upgrade the k8s control plane incrementally through consecutive minor versions, to a maximum of three minor versions
```

(This is getting very verbose but it's a complex topic!)

Reply:
I've made the repeat part its own step in https://github.com/2i2c-org/infrastructure/pull/3420/commits/decdb37c343aa1de6c860668b20c18e6c7bf185a